test/livebook/intellisense_test.exs

```diff
@@ -1180,10 +1180,19 @@ defmodule Livebook.IntellisenseTest do
                Intellisense.get_details("Integer.to_string(10)", 2, context)
     end
 
+    test "returns hexdocs link" do
+      context = eval(do: nil)
+
+      assert %{contents: [hexdocs_link, _]} =
+               Intellisense.get_details("Integer.to_string(10)", 15, context)
+
+      assert hexdocs_link =~ "https://hexdocs.pm/"
+    end
+
     test "does not return duplicate details for functions with default arguments" do
       context = eval(do: nil)
 
-      assert %{contents: [_]} = Intellisense.get_details("Integer.to_string(10)", 15, context)
+      assert %{contents: [_, _]} = Intellisense.get_details("Integer.to_string(10)", 15, context)
```

Comment:
When looking up the function we find both `Integer.to_string/1` and `Integer.to_string/2`, however when there are default arguments, as in `Integer.to_string(integer, base \\ 10)`, we want to show the docs once. Currently we do `Enum.uniq` to eliminate duplicates, but with the hexdoc URL, the docs are no longer the same (they differ in arity in the URL hash). We need to filter out lower-arity functions generated by defaults :)

Reply:
Ignoring arity entirely is not enough, because oftentimes functions with different arity have separate docs. We only want to ignore functions generated by default arguments. This required changes in several places, so I went ahead and did that. I also changed the tuples to maps, because they grew to much :)

Reply:
Thank! It looks good :)